.github/workflows/frontend.yml

```diff
@@ -0,0 +1,22 @@
+name: Frontend
+
+on: [push]
```

Comment:
Let's add pull_request too here.

Reply:
Well spotted, the master branch must suffer from this too.